test/validate_test.rb

```diff
@@ -260,9 +260,10 @@ class ValidateTest < BaseTest
     describe "invalid" do
       before { subject.validate({}).must_equal false }
 
-      it { subject.errors.messages.must_equal(
-        :songs => ["is too short (minimum is 1 characters)"],
-        :hit   => ["can't be blank"]) }
+      it do
+        subject.errors.messages[:hit].must_equal(["can't be blank"])
```

Comment:
We should add a test that no other keys are in `messages` (for completeness). Thanks for your work, can you please add this? :beers: